src/plugins/core.js

```diff
@@ -394,10 +394,28 @@ function Plugin(options = {}) {
     debug('onDropText', { data })
 
     const { text, target } = data
+    const { document } = state
     const transform = state
       .transform()
       .select(target)
 
+    let hasVoidParent = document.hasVoidParent(target.get('anchorKey'))
+
+    // For void targets, we want to find the nearest non-void text node
```

Comment:
Can you clarify this comment to just mention that we'll be inserting the text into the next nearest text node? Right now it's not very clear as written.